cli/azd/resources/apphost/templates/azd.operations.yamlt

```diff
@@ -0,0 +1,13 @@
+{{define "azd.operations.yaml" -}}
+operations:
+{{range $name, $value := .ContainerApps -}}
+{{range $bMount := $value.BindMounts -}}
+- type: FileShareUpload
+  description: Upload files for {{$name}}
```

Comment:
NIT: you might want to include the name of the bind-mount in addition to the name of the service?

Reply:
bindMounts are defined in the manifest w/o a name XD.  That's why I did not mention any name here.
The name is auto-generated by azd as implementation details

![image](https://github.com/Azure/azure-dev/assets/24213737/5a61e6e8-8d27-4215-b885-cc9aeb60fe56)